lua/lazyvim/plugins/extras/lang/java.lua

```diff
@@ -87,8 +87,7 @@ return {
     opts = function()
       local cmd = { vim.fn.exepath("jdtls") }
       if LazyVim.has("mason.nvim") then
-        local mason_registry = require("mason-registry")
-        local lombok_jar = mason_registry.get_package("jdtls"):get_install_path() .. "/lombok.jar"
+        local lombok_jar = LazyVim.get_pkg_path("jdtls", "/lombok.jar")
```

Comment:
```suggestion
        local lombok_jar = vim.fn.expand "$MASON/share/jdtls/lombok.jar"
```

Reply:
I also fixed it this way. Also, using `$MASON` is faster than `LazyVim.get_pkg_path`.